README.md

```diff
@@ -15,6 +15,12 @@ This project should train and test users to use the quicker keyboard shortcuts i
 Our goal for the backend project is to create a persistent app where users 
 can sign in, practice with the app, and save their progress. That way, the next time they sign in, they can pick up where they left off rather than starting at the beginning.
 
+# Setting up Auth0 and creating an account
+1. Create a [free personal Auth0 account](https://auth0.com/)
+1. Create a `Regular Web Application`
+1. Run `mv .env.example .env` to create a .env file
```

Comment:
Can you make this `cp` instead of `mv` otherwise the example will show as a deletion in git?